cli.js

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env node
+'use strict';
+
+var path = require('path');
+var childProcess = require('child_process');
+var inquirer = require('inquirer');
+var assign = require('lodash.assign');
+var utils = require('./cli-utils');
+var codemods = require('./codemods.json');
+
+function runScripts(scripts, files) {
+	var spawnOptions = {
+		env: assign({}, process.env, {PATH: path.resolve('node_modules/.bin') + ':' + process.env.PATH})
+	};
```

Comment:
add option:

```js
stdio: 'inherit'
```

Docs are [here](https://nodejs.org/docs/v0.11.13/api/child_process.html#child_process_child_process_spawn_command_args_options). (You need to scroll way down. ridiculous there isn't a closer anchor for such an important option)